src/main/kotlin/io/curity/identityserver/plugin/frejaeid/authentication/RequestLogicHelper.kt

```diff
@@ -98,48 +112,92 @@ class RequestLogicHelper(private val config: FrejaEidAuthenticatorPluginConfig)
         return emptyList()
     }
 
-    fun getAuthTransaction(postData: Map<String, Any>): Map<String, Any>
+    fun doAuthenticate(postData: Map<String, Any>): Map<String, Any>
```

Comment:
please don't use `doXyz` as names. `authenticate` is a verb and makes for a great name